.ci.yaml

```diff
@@ -2323,6 +2323,8 @@ targets:
       tags: >
         ["devicelab","hostonly"]
       task_name: module_test_ios
+      device_os: iOS-15.1
```

Comment:
If this is expecting a device, we should change it's name from Mac to Mac_ios

Reply:
No, this is now a hostonly test which uses the simulator instead of a real device.

Reply:
In that case, does this need to be added? Looking at it, you'll get a swarming task with:

```yaml
device_type: None # Will run on a VM
device_os: iOS-15.1 # This will be passed, but not be used by swarming. It could be used by the recipe
```